source/dedupsource.go

```diff
@@ -45,7 +45,7 @@ func (ms *dedupSource) Endpoints(ctx context.Context) ([]*endpoint.Endpoint, err
 	}
 
 	for _, ep := range endpoints {
-		identifier := ep.DNSName + " / " + ep.SetIdentifier + " / " + ep.Targets.String()
+		identifier := ep.RecordType + " / " + ep.DNSName + " / " + ep.SetIdentifier + " / " + ep.Targets.String()
```

Comment:
Other options could be
```
identifier := strings.Join([]string{ep.DNSName, ep.SetIdentifier, ep.Targets.String()}, " / ")
or
identifier := fmt.Sprintf("%s / %s / %s", ep.DNSName, ep.SetIdentifier, ep.Targets.String())
```

Do we know why there are empty spaces `" / "`? Just a question?

Reply:
I'm not sure why the spaces are there. Changing it to just "/" doesn't seem to impact any tests but I'd kept that behavior as is just in case.